endpoints/groups.md

```diff
@@ -89,3 +91,37 @@ On success, this returns ```204 No Content```. If the request is invalid, a ```4
 ## Delete Group
 
 * ```DELETE /groups/:id.json``` will delete the specified group and return ```204 No Content``` on success. If the user does not have permission to delete the group, it returns ```404 Not Found```.
+
+## Add Contacts to Group
+
+* ```POST /groups/:id/contacts.json``` has two forms:
+
+### Add Existing Contacts by ids
+
+* ```POST /groups/:id/contacts.json``` adds the contacts specified by the `ids` param to the specified group.
+
+```json
+{
+  "ids": [1, 2, 3]
+}
+```
+
+On success, this returns ```204 No Content```. If the request is invalid, a ```422 Unprocessable Entity``` is returned with the error(s) in the response body. (See [422 Unprocessable Entity](https://github.com/outstand/api-docs/blob/master/422.md))
```

Comment:
If one of the contacts can't be found, do we report that or do we still say `204`?

Reply:
We just ignore it and add any others that are valid.